src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/Client/BlazorHosted-CSharp.Client.csproj

```diff
@@ -13,8 +13,16 @@
     <PackageReference Include="Microsoft.AspNetCore.Blazor.HttpClient" Version="$(TemplateBlazorPackageVersion)" />
   </ItemGroup>
 
+  <!--#if !Hosted -->
   <ItemGroup>
-    <ProjectReference Include="..\BlazorHosted-CSharp.Shared\BlazorHosted-CSharp.Shared.csproj" />
+      <PackageReference Include="Microsoft.AspNetCore.Blazor.DevServer" Version="$(TemplateBlazorPackageVersion)" PrivateAssets="all" />
```

Comment:
Is there any reason to not always include this so that you have the option to run the client project directly?

Reply:
Goal was to have parity between what was already there. If the above is valuable, no issue there